include/tvm/relay/transform.h

```diff
@@ -451,6 +452,100 @@ TVM_DLL Pass ToGraphNormalForm();
  */
 TVM_DLL Pass PartialEval();
 
+/*!
+ * \brief Simplify certain operators during inference. For example, batch norm
+ * will be unpacked into a number of simplified operators.
+ *
+ * \return The Pass.
+ */
+TVM_DLL Pass SimplifyInference();
+
+/*!
+ * \brief Infer the type of an expression.
+ *
+ * The result of type checking is a new expression with unambigous
+ * type information filled in, as well as it's checked type field
+ * populated with the result type.
+ *
+ * \return The pass. 
+ */
+TVM_DLL Pass InferType();
+
+/*!
+ * \brief Search and eliminate common subexpression. For example, if there are
+ * two expressions evaluated to an identical value, a single variable is created
+ * and these two expressions are replaced by this variable.
+ *
+ * \param fskip The callback argument that allows to skip certain expressions.
+ *
+ * \return The pass.
+ */
+TVM_DLL Pass EliminateCommonSubexpr(
+    PackedFunc fskip = PackedFunc([](TVMArgs args, TVMRetValue* rv) {
```

Comment:
This default is a bit long. While we might need this default somewhere. We can just pass in nullptr instead of this function for now. The implementation behavior now could detect fskip=nullptr and run the default function.